src/sparkx/Jetscape.py

```diff
@@ -222,7 +222,32 @@ def _particle_as_list(self, particle: Particle) -> List[Union[int, float]]:
         particle_list[6] = float(particle.pz)
 
         return particle_list
+    
+    def _update_after_merge(self, other: BaseStorer) -> None:
+        """
+        Updates the current instance after merging with another Jetscape instance.
+
+        This method is called after merging two Jetscape instances to update the
+        attributes of the current instance based on the attributes of the other instance.
+        The last line, filename and :code:`sigmaGen` are taken from the left-hand instance.
```

Comment:
Is it possible to implement it in a way the `sigmaGen` is the average value after adding the objects? Then the uncertainty, which is the second element in the tuple should have the according uncertainty due to Gaussian error propagation.

Reply:
Good idea, I gonna do that.

Reply:
Done